ToneMapper/Model/Infastracture/TMGLProgram/Internal Shaders/TMGLIdentityProgram.fsh

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015 Lightricks. All rights reserved.
+// Created by Barak Yoresh.
+
+varying highp vec2 textureCoordinates;
```

Comment:
It is better to start with lowp variables and change them to highp only if you have any errors.